<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1451</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow
{
   background-color: #356a9c;
}

QTabWidget::pane
{
    background-color: #000;
    color: #fff;
    border: 3px solid #80f;
    border-top-right-radius: 10px;
    border-top-left-radius: 5px;
    border-bottom-right-radius: 15px;
    border-bottom-left-radius: 15px;
}

QTabBar::tab
{
    background-color: #000;
    color: #888;
    border: 2px solid #40c;
    padding: 3px;
    border-left-width: 1px;
    border-right-width: 1px;
}

QTabBar::tab:first
{
    border-left-width: 2px;
    border-top-left-radius: 5px;
}

QTabBar::tab:last
{
    border-right-width: 2px;
    border-top-right-radius: 5px;
}

QTabBar::tab:selected
{
    background-color: #444;
    color: #fff;
    border-color: #80f;
}

QLabel
{
    color: #fff;
}

QGroupBox
{
    color: #fff;
    border: 2px solid #40c;
    border-radius: 5px;
}

QPushButton
{
    background-color: #444;
    border: 2px solid #666;
    color: #fff;
    border-radius: 3px;
    padding: 4px;
}

QPushButton:hover
{
    background-color: #666;
    border: 2px solid #aaa;
}

QPushButton:disabled
{
    background-color: #111;
    border: 2px solid #222;
    color: #888;
}

QComboBox
{
    background-color: #444;
    border: 2px solid #666;
    color: #fff;
    border-radius: 3px;
    padding: 4px;
}

QComboBox:disabled
{
    background-color: #111;
    border: 2px solid #222;
    color: #888;
}

QComboBox QAbstractItemView
{
    selection-background-color: #000;
    background-color: #444;
    border: 2px solid #666;
    border-radius: 3px;
    padding-bottom: 2px;
    padding-top: 2px;
    color: #ccc;
}

QDialog
{
    background-color: #000;
}

QRadioButton, QCheckBox
{
    color: #fff;
}

QRadioButton:disabled, QCheckBox:disabled
{
    color: #888;
}

QLineEdit, QSpinBox, QTextEdit, QListWidget
{
    color: #fff;
    background-color: #113;
    border: 2px solid #666;
}

QLineEdit:disabled, QSpinBox:disabled, QTextEdit:disabled, QListWidget:disabled
{
    color: #888;
    background-color: #002;
    border: 2px solid #333;
}

QLineEdit:hover, QSpinBox:hover, QTextEdit:hover, QListWidget:hover
{
    background-color: #114;
    border: 2px solid #666;
}




#label_accuracy, #label_searchPrecision
{
    color: #888;
}

#horizontalSlider_searchPrecision::handle:horizontal
{
    background-color: #333;
}

QTableWidget, QHeaderView::section, QTableWidget QTableCornerButton:section
{
    background-color: #222;
    color: #fff;
    gridline-color: #000;
    alternate-background-color: #111;
}

QTableWidget::item:selected
{
    background-color: #80f;
}

QHeaderView:active
{
    background-color: #222;
    border: 2px solid #000;
}

/*#tableWidget_memory::item:selected
{
    width: 22px;
}*/


/*#tableWidget_memory QHeaderView::section:horizontal
{
    width: 22px;
}*/</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>1031</width>
      <height>711</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Người đăng</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mã tài liệu</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tài liệu môn</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ngày đăng</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ghi chú</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Đọc</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thêm vào yêu thích</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>212</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="xem_favor_tai_lieu">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
font: 600 16pt &quot;Bahnschrift&quot;;
background-color:rgb(129, 201, 255);
color: #F8F2EC;</string>
       </property>
       <property name="text">
        <string>Xem tài liệu yêu thích</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upload_tai_lieu">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
font: 600 16pt &quot;Bahnschrift&quot;;
background-color:rgb(129, 201, 255);
color: #F8F2EC;</string>
       </property>
       <property name="text">
        <string>Upload tài liệu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
